projects/experimental/components/badge/badge.style.less

```diff
@@ -0,0 +1,461 @@
+        color: var(--tui-primary-text);
+    }
+
+    &[data-appearance='default'] {
+        background-color: var(--tui-base-06);
+
+        &:not([data-mode]) {
+            &._hoverable:hover {
+                background-color: var(--tui-base-07);
+            }
+
+            &._hoverable:active {
+                background-color: var(--tui-base-08);
+            }
+        }
+
+        &[data-mode='onDark'] {
+            background-color: var(--tui-clear-inverse-hover);
+
+            &._hoverable:hover {
+                background-color: var(--tui-clear-inverse-active);
+            }
+
+            &._hoverable:active {
+                background-color: var(--tui-clear-inverse-active);
+            }
+
+            &:focus:after {
+                border: 2px solid var(--tui-base-01);
+            }
+        }
+
+        &[data-mode='onLight'] {
+            background-color: var(--tui-clear);
+            color: var(--tui-text-01);
+
+            &._hoverable:hover {
+                background-color: var(--tui-clear-hover);
+            }
+
+            &._hoverable:active {
+                background-color: var(--tui-clear-active);
+            }
+        }
+    }
+
+    &._hoverable {
+        cursor: pointer;
+    }
+
+    &[data-padding='m'] {
+        padding: 0;
+    }
+
+    &[data-padding='l'] {
+        padding: 0 0.25rem;
+
+        &[data-size='xl'] {
+            padding: 0 0.375rem;
+        }
+
+        &[data-size='xl'][data-platform='web'] {
+            padding: 0 0.5rem;
+        }
+
+        &[data-size='s'] {
+            padding: 0 0.125rem;
+        }
+
+        &[data-size='s'][data-platform='web'] {
+            padding: 0 0.25rem;
+        }
+    }
+
+    &[data-size='s'] {
+        height: 1.25rem;
+        min-width: 1.25rem;
+
+        .t-left-content:before {
+            width: 0.25rem;
+            height: 0.25rem;
+        }
+
+        &[data-platform='ios'],
+        &[data-platform='android'] {
+            height: 1rem;
+            min-width: 1rem;
+        }
+    }
+
+    &[data-size='m'] {
+        height: 1.25rem;
+        min-width: 1.25rem;
+
+        .t-left-content:before {
+            width: 0.25rem;
+            height: 0.25rem;
+        }
+    }
+
+    &[data-size='l'] {
+        height: var(--tui-height-xs);
+        min-width: var(--tui-height-xs);
+
+        &[data-platform='ios'],
+        &[data-platform='android'] {
+            height: 1.5rem;
+            min-width: 1.5rem;
+        }
+    }
+
+    &[data-size='xl'] {
+        height: var(--tui-height-s);
+        min-width: var(--tui-height-s);
+
+        &[data-platform='ios'],
+        &[data-platform='android'] {
+            height: 1.75rem;
+            min-width: 1.75rem;
+        }
+    }
+
+    &._empty-value {
+        .t-left-content {
+            margin: 0 -0.25rem;
+
+            &:before {
+                margin: 0;
+            }
+        }
+    }
+}
+
+.t-text {
+    font: var(--tui-font-text-s);
```

Comment:
Can we move this on host? So if somebody would like to make text in all badges bold they can do it easily.